examples/react/.babelrc

```diff
@@ -1,17 +0,0 @@
-{
-  "presets": [
```

Comment:
The idea was to show in this example that Gondel can be used without typescript as well